coinstacks/bitcoin/api/src/controller.ts

```diff
@@ -58,23 +58,44 @@ export class Bitcoin extends Controller implements BaseAPI, BitcoinAPI {
     try {
       let data: Address | Xpub
       if (isXpub(pubkey)) {
-        data = await blockbook.getXpub(pubkey, undefined, undefined, undefined, undefined, 'tokenBalances', 'used')
+        data = await blockbook.getXpub(pubkey, undefined, undefined, undefined, undefined, 'txs', 'used')
```

Comment:
No need to fetch verbose transactions anymore
```suggestion
        data = await blockbook.getXpub(pubkey, undefined, undefined, undefined, undefined, 'tokenBalances', 'used')
```